public/app/features/dashboard/state/DashboardMigrator.test.ts

```diff
@@ -66,6 +66,7 @@ describe('DashboardModel', () => {
         panels: [
           {
             type: 'graph',
+            // @ts-expect-error
```

Comment:
I took the liberty of moving this ts-expect-error here because I saw two errors from the typecheck:
```
public/app/features/dashboard/state/DashboardMigrator.test.ts:67:11 - error TS2578: Unused '@ts-expect-error' directive.

67           // @ts-expect-error
             ~~~~~~~~~~~~~~~~~~~

public/app/features/dashboard/state/DashboardMigrator.test.ts:70:13 - error TS2322: Type 'boolean' is not assignable to type '{ show: boolean; sort?: string | undefined; sortDesc?: boolean | undefined; }'.

70             legend: true,
               ~~~~~~
```

but now I sometimes see a fully successful pass of `yarn typecheck` and build the frontend successfully, and sometimes see some errors... I hope everything is alright, but please check one last time before you merge.

Reply:
I experienced this problem as well. it seems to be a problem of how TS updates the cached results. If you remove from your local `tsconfig.tsbuildinfo` you will see the type check success deterministically.